Automattic/wp-calypso
# Plans Page: Adds estimate for Search

#### Changes proposed in this Pull Request

* This adds a disclaimer to Jetpack Search that the number of records are an estimate only.

#### Screenshots
<img width="356" alt="Screen Shot 2021-01-07 at 3 49 36 PM" src="https://user-images.githubusercontent.com/1760168/103943495-2e165f00-5100-11eb-93f3-02efb80500a8.png">
<img width="350" alt="Screen Shot 2021-01-07 at 3 50 59 PM" src="https://user-images.githubusercontent.com/1760168/103943493-2e165f00-5100-11eb-9ef7-e419d9fb3bce.png">

#### Testing instructions



1. View the plans page for a site with Jetpack.
2. Observe Jetpack Search card shows the estimate disclaimer with correct number of records from API.
3. Verify prices page with no site is identical to #48698.

Fixes 1164141197617539-as-1199519139839503.
